birdcage_frontend/templates/birdsoftheweek.html

```diff
@@ -1,11 +1,11 @@
 {% extends "base.html" %}
 
 {% block title %}
-Vos oiseaux de la semaine
+Birds of the Week
 {% endblock %}
 
 {% block content %}
-<h1>Vos oiseaux de la semaine</h1>
+<h1>Birds of the Week</h1>
```

Comment:
Forgot to code switch I guess here? :D

Reply:
Nah I just like the word oiseaux - one of the few things I remember from taking French.  I'll probably change it back :)